book/installation/docker.md

```diff
@@ -125,3 +125,13 @@ docker exec -it reth bash
 **If Reth is running with Docker Compose, replace `reth` with `reth-reth-1` in the above command**
 
 Refer to the [CLI docs](../cli/cli.md) to interact with Reth once inside the Reth container.
+
+## Run only Grafana in Docker
+
+This allows importing existing Grafana dashboards, without running reth in Docker.
+
+```bash
+docker compose -f etc/docker-compose.yml up -d --no-deps grafana
+```
+
+After login with `admin:admin` credentials, this should list Prometheus under [`Grafana datasources`](http://localhost:3000/connections/datasources). Replace its `Prometheus server URL` with `http://host.docker.internal:9090`.
```

Comment:
I believe this `host.docker.internal` thing works only on Mac, for Linux see https://stackoverflow.com/questions/48546124/what-is-the-linux-equivalent-of-host-docker-internal